bin/templates/test.yaml.j2

```diff
@@ -0,0 +1,28 @@
+{#
```

Comment:
Can we have a more descriptive name for this file?   Ideally it should be obvious from the filename that this is running the embench tests.

Reply:
Yes, this is possible. The reason it's originally named "test.yaml.j2" is that it matches the name of the file(s) that are generated from this template, "test.yaml". If you prefer the naming to match the use, say "embench_test.yaml.j2", that is an easy fix.